### PR TITLE
fixes #2053

### LINKED ISSUE #2053: No mostrar "empieza a usar escuelas" para usarios que ya lo usan
Si ya se está dentro de un curso ya no debería de aparecer la pantalla de “empieza a usar escuelas” y te debería dirigir directamente a tus cursos.

### PR DESCRIPTION
**Pull request by sarvesh211999**
This PR fixes issue #2053 .
index.php in www/schools is changed 
only if else condition is added

### CODE REVIEW

File: frontend/www/schools/index.php [hunk 1]
```diff
@@ -1,4 +1,12 @@
 <?php
 
 require_once('../../server/bootstrap.php');
-$smarty->display('../templates/schools.intro.tpl');
+$currentSession = SessionController::apiCurrentSession()['session'];
```
Comment (pabloaguilar) [inline frontend/www/schools/index.php]: You're not using the current session

**Comment by sarvesh211999** [comment]
@lhchavez review ??

**Comment by heduenas** [comment]
@lhchavez is on vacation right now. It might take him some extra time to answer. Thanks for contributing, btw.

**Comment by sarvesh211999** [comment]
Okay

**Comment by pabo99** [comment]
Maybe you need to fix the automated test in `Selenium` because your change is modifying the navigation.

This is unnecesary because you are redirecting in `www/schools/index.php`
https://github.com/omegaup/omegaup/blob/73d9541aa6ee99addea93467f3dd45ab4b4c9b43/frontend/tests/ui/test_course.py#L89-L92

**Comment by pabo99** [comment]
Same case in L212, but be careful to review if it's first time user accesses to Schools

**Comment by sarvesh211999** [comment]
@lhchavez review

**Pull request merged**